docs/03-process.md

```diff
@@ -3,8 +3,83 @@ title: Design Process
 label: Process
 status: draft
 ---
-## Design Process
+## College Design Language System (CDLS) Component Update/Addition Request 
 
-Governance process for accepting additions/changes to the College DLS.  To be determined after our base set of deliverables from Simpson Scarborough are in a stable state (release 1).
+### Introduction
+This document outlines the design process for adding a new component to the UTSA College Design Language System (CDLS). The objective is to seamlessly update an existing component or integrate the new component into the established design system, ensuring consistency and cohesiveness with the existing components.
 
-### More detail to follow
+### Design Process Overview 
+The design process for adding a new component involves the following steps:
+<ol>
+    <li><strong>Design Request</strong>
+        <ul>
+            <li>Stakeholder submits request to the web team through online form (https://provost.utsa.edu/communications/) or by communicating directly with web team through Teams</li>
```

Comment:
Looks great! I would make "online form", a link, or the provost website a link. To do this in Markdown you would use the following code: 
`[Link Name](Link URL)`